ddtrace/contrib/pynamodb/__init__.py

```diff
@@ -0,0 +1,26 @@
+"""
```

Comment:
We should also update `docs/index.rst` to include pynamodb and the supported versions